cluster-scope/overlays/ocp-prod/kustomization.yaml

```diff
@@ -22,6 +22,7 @@ resources:
   - ../../base/operators.coreos.com/subscriptions/kubernetes-nmstate-operator
   - ../../base/operators.coreos.com/subscriptions/ocs-operator
   - ../../base/operators.coreos.com/subscriptions/openshift-nfd
+  - ../../base/operators.coreos.com/subscriptions/vertical-pod-autoscaler
   - ../../base/rbac.authorization.k8s.io/clusterrolebindings/cluster-admins-rb
   - ../../base/rbac.authorization.k8s.io/clusterrolebindings/moc-kubernetes-external-secrets
   - ../../base/rbac.authorization.k8s.io/clusterrolebindings/moc-kubernetes-external-secrets-auth
```

Comment:
You also need to include your namespace configuration here (`../../base/core/namespaces/cs6620-fall21-deployverticalpod/`), and you'll need to do the same thing for the `openshift-vertical-pod-autoscaler` namespace when you add those manifests.